VRCFaceTracking/SDK/Attributes.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VRCFaceTracking
+{
+    /*
+    public enum VRCFTModuleType
+    {
+        Eye,
+        Expression,
+        Both
+    }
+
+    /// <summary>
+    /// Specifies the module type info relevant for VRCFaceTracking.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly)]
+    public class VRCFTModuleTypeAttribute : Attribute
+    {
+        /// <summary>
+        /// Container of the type of data the module is expected to send.
+        /// </summary>
+        public VRCFTModuleType ModuleType { get; private set; }
+
+        public VRCFTModuleTypeAttribute(VRCFTModuleType moduleType)
+        {
+            ModuleType = moduleType;
+        }
+    }
+    */
+
+    /// <summary>
+    /// Specifies loading this module prior to the specified module GUID.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly)]
+    public class VRCFTModuleEnsureLoadPriorAttribute : Attribute
+    {
+        /// <summary>
+        /// Container of the referenced module GUID.
+        /// </summary>
+        public string ModuleGUID { get; private set; }
+
+        public VRCFTModuleEnsureLoadPriorAttribute(string moduleGUID)
+        {
+            ModuleGUID = moduleGUID;
+        }
+    }
+
+    /// <summary>
+    /// Specifies loading this module directly after the specified module GUID.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly)]
+    public class VRCFTModuleEnsureLoadAfterAttribute : Attribute
+    {
+        /// <summary>
+        /// Container of the referenced module GUID.
+        /// </summary>
+        public string ModuleGUID { get; private set; }
+
+        public VRCFTModuleEnsureLoadAfterAttribute(string moduleGUID)
+        {
+            ModuleGUID = moduleGUID;
+        }
+    }
```

Comment:
Just want to double check and make sure that these attributes are what we expect them to be? VRCFTModuleEnsureLoadAfter will load after a specified module GUID, and VRCFTModuleEnsureLoadPriorAttribute will load before a specified module GUID. This affects the order they appear in the ListBox on the UI, and how they are initialized by default. I believe I have it so that the user order will override this but I can't exactly remember if I allowed that interaction or not.

Reply:
Note that EnsureLoadAfter is currently unimplemented, EnsureLoadPrior was what @benaclejames was wanting implemented for specific modules.